airbyte-cdk/java/airbyte-cdk/core/src/main/kotlin/io/airbyte/cdk/db/jdbc/JdbcUtils.kt

```diff
@@ -62,7 +62,7 @@ object JdbcUtils {
             JDBCType.VARCHAR,
             JDBCType.LONGVARCHAR
         )
-    @JvmField val defaultSourceOperations: JdbcSourceOperations = JdbcSourceOperations()
+    @JvmStatic val defaultSourceOperations: JdbcSourceOperations = JdbcSourceOperations()
```

Comment:
@jdpgrailsdev  This was accessed as a static member in Java, any better way to do than the escape hatch ?

Reply:
not sure what the escape hatch is